src/Microsoft.VisualStudio.Web.CodeGeneration.Core/Microsoft.VisualStudio.Web.CodeGeneration.Core.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\dir.props" />
+  <PropertyGroup>
+    <Description>Contains the core infrastructure used by ASP.NET Core Code Generators.</Description>
+    <TargetFrameworks>net451;netstandard1.6</TargetFrameworks>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
```

Comment:
I didn't see `<GenerateDocumentationFile>true</GenerateDocumentationFile>` set anywhere to ensure xml docs are produces.